data/transactions/logic/Makefile

```diff
@@ -7,15 +7,13 @@ DOCS :=  ../../../../docs
 
 TEAL_opcodes.md teal.tmLanguage.json:	fields_string.go ../../../cmd/opdoc/opdoc.go ../../../cmd/opdoc/tmLanguage.go eval.go assembler.go doc.go opcodes.go
 	go run ../../../cmd/opdoc/opdoc.go ../../../cmd/opdoc/tmLanguage.go
+	@cp `ls TEAL_opcodes_v??.md | tail -1` TEAL_opcodes.md
```

Comment:
So this concatenates all files into a single `TEAL_opcodes.md`? Is there a deterministic order?

Reply:
No, the `tail -1` takes the last one. And we only `ls` the ones with two digits because of `v??`.  So it's just to temporarily call the final version `TEAL_opcodes.md` (we delete it a couple lines later).
This was easier (for me!) than figuring out how to use shell variables inside the Makefile which itself has `$` variables.

Reply:
Thanks for the explanation, that makes sense.